### PR TITLE
Add assertion for HTML equality

### PR DESCRIPTION
**Pull request by NoahCarnahan**
Add a method, `assertHtmlEqual`, that mirrors `assertXmlEqual` but parses the markup as HTML.

### CODE REVIEW

File: corehq/apps/app_manager/tests/util.py [hunk 1]
```diff
@@ -70,11 +63,33 @@ def normalize_attributes(xml):
...
+        meth = "html"
+    parser = parser_class(remove_blank_text=True)
+    parse = lambda *args: normalize_attributes(markup_class(*args))
     parsed = parse(xml, parser)
-    return lxml.etree.tostring(parsed, pretty_print=True) if to_string else parsed
+    return lxml.etree.tostring(parsed, pretty_print=True, method=meth) if to_string else parsed
+
+
+def _check_shared(expected, actual, checker, extension):
+     # snippet from http://stackoverflow.com/questions/321795/comparing-xml-in-a-unit-test-in-python/7060342#7060342
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/util.py]: E111 indentation is not a multiple of four

Comment (dimagimon) [inline corehq/apps/app_manager/tests/util.py]: E501 line too long (116 > 115 characters)

**Comment by snopoke** [comment]
Looks good, waiting for build.

**Comment by NoahCarnahan** [comment]
Looks like it passed. Thanks!

**Pull request merged**